content/en/docs/languages/python/libraries.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Using instrumentation libraries
+linkTitle: Libraries
+weight: 40
+cSpell:ignore: HTTPX httpx instrumentor uninstrument
+---
+
+{{% docs/languages/libraries-intro "Python" %}}
+
+## Use instrumentation libraries
+
+If a library isn't included with OpenTelemetry, you can use
+[instrumentation libraries](/docs/specs/otel/glossary/#instrumentation-library)
+to generate telemetry data for a library or framework.
+
+For example,
+[the instrumentation library for HTTPX](<[https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNetCore](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation/opentelemetry-instrumentation-httpx)>)
```

Comment:
We can get rid of the nuget link :)

Reply:
```suggestion
[the instrumentation library for HTTPX](https://pypi.org/project/opentelemetry-instrumentation-httpx/)
```

Also point to pypi packages, this is where Python developers will look for it